src/pages/interviews/serverlesschats.mdx

```diff
@@ -17,3 +17,124 @@ On this episode, Jeremy and Rebecca chat with Swizec Teller about how to approac
+
+"my ideal Serverless server is basically a JavaScript function written in node.js that accepts some arguments." - ST
+
+"some people don't realize this, but your database will work just fine if column names are in camel case.6 It's going to look weird and you might shriek in horror when you see that. But the database doesn't care. the database it's just a string. Same with JavaScript. If you do some snake case in JavaScript, it's just, it's fine. It's just a string. It doesn't care what you call your variables." - ST
+
+"the nice thing with serverless is that it abstracts away all of the dev ops stuff. And all of the wiring just happens on,, essentially automatically on the infrastructure of AWS or whatever provider you're using. And you can focus on writing the business logic." - ST
+
+"If you're using something like edge workers that are becoming more and more popular, it will find the nearest server, the nearest data center to the user, wakes up, a server runs all of the wiring sets up all of the frameworky stuff. Calls your function with a parsed object that just has the arguments that you want." - ST
+
+"the biggest thing that, especially front-end engineers coming to serverless don't realize is that you can get huge latencies and slowness if you get it wrong. And by get it wrong I mean, the biggest, like the Achilles heel of something like Versal or Netlify functions or even Gatsby functions is when, as soon ,as you have to talk to a database, because you can't configure the database in the same environment." - ST
+
+"it feels like, 'Oh, if I add Postgres to my thing, or if I add Mongo, it's suddenly it's slow. It's not actually slow. It's just the connection takes awhile." - ST
+
+"when in high school I was installing a Linux and I got it wrong. And because I was just following a tutorial, I didn't know what I was doing. So two days into compiling, gentu, on my very slow computer back then. I got it wrong and I had to start from scratch. And yes, it actually took me three days to install a new operating system. Those were the days. And I'm happy we're not there anymore." - ST
+
+"personally really liked the serverless framework because it's a really good middle ground between all of the pain of cloud formation and all of the abstraction of something like Netlify or Vercel or Gatsby or whatever. Because it gives you all of the power, but it makes the simple things easy" - ST
```

Comment:
```suggestion
"personally really liked the serverless framework because it's a really good middle ground /../ it gives you all of the power, but it makes the simple things easy" - ST
```